components/NavigationBar.vue

```diff
@@ -1,33 +1,36 @@
 <template>
-  <div :class="classes" class="k-navigation-bar">
-    <div class="k-navigation-bar__placeholder" />
-    <nav class="k-navigation-bar__container-0">
-      <div class="k-navigation-bar__toggle" @click="open = !open">
+  <div class="navigation-bar" :class="{ 'x-scrolled': scrolled, 'x-show-background': showBackground, 'x-open': open }">
+    <div class="navigation-bar__placeholder" />
+    <nav class="navigation-bar__container-0">
+      <div class="navigation-bar__toggle" @click="open = !open">
         <span />
         <span />
         <span />
       </div>
-      <div class="k-navigation-bar__container-1 content">
-        <component
-          :is="isNuxt ? 'nuxt-link' : 'router-link'"
-          :key="'cryptic-logo'"
-          :to="'/'"
-          @click.native.passive="open = false"
-        >
-          <CrypticHeadLogo class="k-navigation-bar__logo" />
-        </component>
-        <span class="k-navigation-bar__title">{{ title }}</span>
-        <div class="k-navigation-bar__container-2">
-          <div class="k-navigation-bar__links">
+      <div class="navigation-bar__container-1 content">
+        <div class="navigation-bar__logos">
+          <component
+            :is="isNuxt ? 'nuxt-link' : 'router-link'"
+            :key="'cryptic-logo'"
+            :to="'/'"
+            @click.native.passive="open = false"
+          >
+            <CrypticHeadLogo class="navigation-bar__logo" />
+          </component>
+          <LanguageSwitcher class="navigation-bar__language" />
+        </div>
+        <span class="navigation-bar__title">{{ title }}</span>
```

Comment:
translate the title -> 
`<span class="k-navigation-bar__title">{{ $t("navbar." + title) }}</span>`